simple_settings/dynamic_settings/database_reader.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+from sqlalchemy import engine_from_config, Column, String
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
```

Comment:
In redis and consul reader I added a simple try..except to raise ImportError with a better message, you can do this in this file?

https://github.com/drgarcia1986/simple-settings/blob/master/simple_settings/dynamic_settings/redis_reader.py#L4